### ISSUE TITLE
Battlefield 1 (1238840)

### ISSUE THREAD

**Issue by RokasBal** [ValveSoftware/Proton#5047]
# Compatibility Report
- Name of the game with compatibility issues: Battlefield 1
- Steam AppID of the game: 1238840

## System Information
- GPU: 1660 Super
- Driver/LLVM version: NVIDIA 470.57.02
- Kernel version: 5.10.53-1-MANJARO
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/MarioDZN/0448951d76c7cf4466748df004122c32
- Proton version: 6.4-5 / Experimental / 6.14-GE-2 ( Unofficial )

## I confirm:
- [ X] that I haven't found an existing compatibility report for this game.
- [ X] that I have checked whether there are updates for my system available.

[steam-1238840.log](https://github.com/ValveSoftware/Proton/files/6941174/steam-1238840.log)

## Symptoms 

The game seems to launch fine, however the mouse is locked to the center from the very start the cursor appears in the game window. I can move the mouse slightly, however it seems to get sent back to the center of the screen every frame. This makes it very difficult to navigate the menus as the only solution is to use the keyboard for navigation. 

The mouse works once in game, however is once again locked to the center once in the pause menu or death screen. I'm able to exit the pause screen and I can aim / play fine, however as mentioned, it's difficult not being able to use the menus.

**Comment by Klairm** [comment]
Does this have relation with Origin overlay? Maybe if you disable it could fix the problem

**Comment by RokasBal** [comment]
> 
> 
> Does this have relation with Origin overlay? Maybe if you disable it could fix the problem

I had tried disabling the Origin overlay, however it didn't make a difference for this issue.

**Comment by Klairm** [comment]
> > Does this have relation with Origin overlay? Maybe if you disable it could fix the problem
> 
> I had tried disabling the Origin overlay, however it didn't make a difference for this issue.

Do you have 2 monitors or more?

**Comment by RokasBal** [comment]
Yes, that's something that I forgot to mention originally. I have 2 displays. My main display is 21:9 running at 2560 x 1080 and my second display, on the left is 1280 x 1024. I had tested it by unplugging my second display before launching the game and it didn't seem to make a difference.

**Comment by Klairm** [comment]
> Yes, that's something that I forgot to mention originally. I have 2 displays. My main display is 21:9 running at 2560 x 1080 and my second display, on the left is 1280 x 1024. I had tested it by unplugging my second display before launching the game and it didn't seem to make a difference.

Have you tried different screen configurations? I have seen this happens on Windows too and it solves changing from fullscreen to windowed .

Also try this -> https://answers.ea.com/t5/Battlefield-1/BF1-Mouse-cursor-issue/m-p/5732943/highlight/true#M7586

**Comment by kisak-valve** [comment]
EA app still crashes as of 2022/11/04 with Proton Experimental (Battlefield 1)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6289.
@hustin-paul posted on 2022-11-04T20:35:10:

# Compatibility Report
- Name of the game with compatibility issues: Battlefield 1 (EA App)
- Steam AppID of the game: 1238840

## System Information
- GPU: RTX 3070 Ti
- Driver/LLVM version: Nvidia 520.56.06
- Kernel version: 6.0.6-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/): [Gist](https://gist.github.com/hustin-paul/7ba7769635f2af783c88bdb4b0177f22)
- Proton version: Experimental 

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



## Symptoms 


## Reproduction

The Proton log is 1.1 GB, compressed is 75 MB. Over the 25 MB limit.

Edit: This was with the recent [Experimental Version](https://github.com/ValveSoftware/Proton/wiki/Changelog)

- Launch game
- EA app flickers on and off screen about 3-5 times
- EA app minimizes to a wine taskbar at the top left
- Click EA icon, EA app shows up
- Clicking on _anything_ results in a crash

**Comment by BETLOG** [comment]
EA App seems to be a fatal issue.
I have Battlefield 4 and V running, both of which use the same EA app in their respective prefixes, but after doing a basic install of Battlefield 1 it just locks up on a blank link2EA screen.
Tried not specifying proton. 
Tried GE-Proton7-38.
![2022-11-10--18-06-22_betlogbeast_Link2EA](https://user-images.githubusercontent.com/12265451/201038238-e9c6d3ef-f3da-49a6-a51b-267197e6652f.png)

**Comment by ms178** [comment]
I can confirm the issue with Battlefield 1, even with latest proton-experimental, the game would not launch any longer. For a couple of days I got it to work with Proton-GE 7.38 but not with any newer release. And even with Proton-GE 7.38 it needed more than one try to actually get into the game only seeing a couple of gray splash screens for the EA app. But that also stopped working yesterday, hence BF1 is now totally broken.

I am on an Arch-derivative with Kernel 6.0.8 and Mesa-git.

**Comment by Thomas-Driessen** [comment]
Can also confirm the issue. Worked fine with proton before for a few times, but as of late it doesn't start at all anymore with any proton version.

Ubuntu 22.10, 5.19.0-23-generic, X11

[System information](https://gist.github.com/Thomas-Driessen/3882564b8f3bf00f3c4446d834a9d151)
[steam-1238840.log](https://github.com/ValveSoftware/Proton/files/10006391/steam-1238840.log)

**Comment by Sloofy** [comment]
Been having issues with the EA Launcher not working on a fresh Battlefield 1 install. Screenshot below appears after first-time setup has finished. Using the latest Proton Experimental version as of November 29th, 2022. Running Fedora 37 KDE with Wayland.

Notably been seeing a lot of "ipv6_forward_enumerate_all not implemented" and "list_manager_GetConnectivity 0000000012750050, 0000000012B9FB2C" (and related) messages. Not sure how important that might be. Let me know if I should provide more detail.

![image](https://user-images.githubusercontent.com/18003771/204714193-99d6640a-ab8c-4958-9bbb-13b5850214dd.png)

[steam-1238840.log](https://github.com/ValveSoftware/Proton/files/10119757/steam-1238840.log)

**Comment by alasky17** [comment]
@Sloofy and anyone else still seeing trouble with the EA launcher + Battlefield 1.  Unfortunately we've been unable to reproduce the failures so far - I just tested with Wayland+KDE and saw no issues.  Could you please try deleting the prefix (located by default in ~/.steam/steam/steamapps/compatdata/1238840) and trying again with Proton Experimental?  Please note - the game can take a while to load, so please wait *at least* 3 minutes after starting to launch.

**Comment by ms178** [comment]
@alasky17 I've just installed the game again, with a four day old proton-cachyos which is based on proton-experimental from the same day. While the EA app seems to install fine, the game still does not start up once the installation process is finished. This is on CachyOS (Arch derivative), Kernel 6.0.10, KDE 5.26.4, X11-git and Mesa-git. Steam itself also got an update from December 2nd.

I've also tried to add the EADesktop.exe and open it as a steam-foreign application but it also simply refuses to load up properly.

**Comment by Sloofy** [comment]
> @Sloofy and anyone else still seeing trouble with the EA launcher + Battlefield 1. Unfortunately we've been unable to reproduce the failures so far - I just tested with Wayland+KDE and saw no issues. Could you please try deleting the prefix (located by default in ~/.steam/steam/steamapps/compatdata/1238840) and trying again with Proton Experimental? Please note - the game can take a while to load, so please wait _at least_ 3 minutes after starting to launch.

@alasky17 I've deleted the prefix a few times now, waiting after the blank EA launcher for anywhere between 5 minutes to an hour. Same results. Still running Proton Experimental. 

I experimented with disabling IPv6 and it does prompt for a few more things, such as a background instance not running and requesting for an update to EADesktop.exe. It'll open a command prompt letting me know the program is already up to date and go back to the blank screen again. Included a new Steam log with that and system information.

[steam-1238840.log](https://github.com/ValveSoftware/Proton/files/10144619/steam-1238840.log)
[system-info.txt](https://github.com/ValveSoftware/Proton/files/10144632/system-info.txt)

**Comment by ms178** [comment]
@kisak-valve What's the status on this? Another two weeks passed by without any news or an ETA for a fix while users are left with a broken game since early November.

**Comment by ms178** [comment]
For anyone reading this, I got some success with the official Proton 7.0-5 albeit the EA app still is not showing up correctly, it at least works good enough to get into the game.

**Comment by BETLOG** [comment]
Is working again.
Deleted compatdata.
Started steam, selected GE-Proton7-42, started game.
Seemed to work well enough in intro movies and initial singleplayer game segment. Didn't try mulptiplayer because all my keybinds are for arrowkeys, not WASD, side of keyboard. Quit to restore them. EAapp crashed after successful game exit.
Seems like the game is back in action.

**Comment by ms178** [comment]
I was asked to provide more info and produced a gist with the Proton-Experimental log and my system information: https://gist.github.com/ms178/a9500519e47f3db8386da40468f86571

Proton 7.0-5 is the last working release for me, neither Proton-GE nor the official Proton-Experimental nor the distro-provided Proton-Cachyos (which is based on Proton-Experimental) do work with this game on my CachyOS installation. For reference, here is a log with the last working 7.0-5: https://1drv.ms/u/s!Agwwh-axGk6DhjgycEXmPBxV3-xT?e=y1Rjc1 

Even when using the trick from here (https://steamdeckhq.com/tips-and-guides/fixing-ea-play-blank-screen-for-ea-games-on-steam/), I only get the EA blank screen issue fixed on Proton 7.0-5.

Note: Due to compile errors, I compile a minimal Mesa without swrast and svga, only radeonsi and radv.

**Comment by ms178** [comment]
**Update**: For the record, as discussed in the Proton 7.0-6 RC thread which now omits some relevant parts of the discussion, my issues could be solved by compiling Mesa-git with SWRAST and without LTO, as there are Mesa build issues with LTO and SWRAST enabled. The Proton 7.0-6 RC introduced a hard dependancy on SWRAST for the EA app and I did not build that Mesa component before.

While this Proton change was labeled as a HACK specifically, I hope that the EA app issue can be fixed in a less invasive way in the future as it regresses X11 users without SWRAST, added a hard dependancy on building Mesa with SWRAST.

**Comment by kisak-valve** [comment]
Battlefield 1 not launching at all

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6516.
@Kidplayer666 posted on 2023-02-10T16:03:06:

# Compatibility Report
- Battlefield 1
- Steam AppID of the game: 1238840

## System Information
- GPU: GTX 750Ti
- Driver/LLVM version: Nvidia 525.85.05
- Kernel version: 6.1.9-1
- Link to full system information report as [Gist](https://gist.github.com/Kidplayer666/95fdf7541223f4b64fb47e5d83b84689):
- Proton version: Attempted to use both proton experimental and the latest GE
- 
## I confirm:
- [ x] that I haven't found an existing compatibility report for this game.
- [ x] that I have checked whether there are updates for my system available.


[steam-1238840.log](https://github.com/ValveSoftware/Proton/files/10709113/steam-1238840.log)

## Symptoms 
I have this game on Steam, and it ran perfectly on windows. I've now tried installing it on Linux, but it simply does not run. I've tried the recommended suggestions on protondb.com, this guide [here](https://www.reddit.com/r/SteamDeck/comments/tx6srv/guide_installing_origin_on_the_steam_deck_the/) using proton GE and proton Experimental with a combination of the following commands 
`DXVK_ASYNC=1 %command%
STEAM_COMPAT_DATA_PATH="/home/MYUSERNAME/.local/share/Steam/steamapps/compatdata/3955941937" gamemoderun %command%'
MANGOHUD=1 DXVK_HUD=compiler RADV_PERFTEST=gpl gamemoderun %command%
DXVK_ASYNC=1 %command%
STEAM_COMPAT_DATA_PATH="STEAM_COMPAT_DATA_PATH="/home/MYUSERNAME/.local/share/Steam/steamapps/compatdata/3955941937/pfx/drive_c/Program Files/Electronic Arts/EA Desktop/EA Desktop/" gamemoderun %command%'`

**Comment by kisak-valve** [comment]
Hello @Kidplayer666, `wine: '/mnt/D6AE7B6BAE7B42D3/SteamLibrary/steamapps/compatdata/1238840/pfx' is not owned by you` in your log is a permissions issue. Proton (and wine) requires for the wineprefix to be owned by the current user running the game to avoid a category of subtle and hard to diagnose issues.

**Comment by Kidplayer666** [comment]
How can I give them the permission though? I should be the only user in the computer (apart from root maybe) so that problem should be unexistant @kisak-valve ?

**Comment by salixor** [comment]
I am experiencing some artifacting on reflections. Does anyone else experience those ? (Not easy to show on screenshots, but I think they are still quite visible).

![Screenshot_20230618_212340](https://github.com/ValveSoftware/Proton/assets/12085737/70ec6279-d351-4c67-acd9-bddd20aec199)
![Screenshot_20230618_211132](https://github.com/ValveSoftware/Proton/assets/12085737/61646fe0-aac4-44b5-89c1-3a33f3c0efb6)

**Comment by kisak-valve** [comment]
Hello @salixor, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Also, please note the Proton version you're running the game with.

**Comment by salixor** [comment]
I'm running Proton experimental.
System info : https://gist.github.com/salixor/87ac8ea067f9754724a786f12a1f1858

**Comment by salixor** [comment]
Here's a way clearer screenshot I managed to grab, just in case it helps !

![Screenshot_20230924_170940](https://github.com/ValveSoftware/Proton/assets/12085737/d458f369-d2b4-45ea-afa8-2d41bbd17dc4)

**Comment by ayanami932** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5047#issuecomment-1732594734

PROTON_ENABLE_NVAPI=1 does fix those for me. The same goes for Battlefield V

**Comment by salixor** [comment]
Can confirm this does work ! Thanks a lot.

**Comment by kisak-valve** [comment]
Battlefield 1 - seeing scope glint through objects

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7643.
@d3vilguard posted on 2024-04-07T16:03:06:

# Compatibility Report
- Name of the game with compatibility issues: Battlefield 1
- Steam AppID of the game: 1238840

## System Information
- GPU: RX 6800
- Video driver version: mesa-minimal-git-24.1.0_devel.187172.7c0b19a6071-1, compile against llvm-git
- Kernel version: 6.8.3 TKG BORE
- Link to full system information report as [Gist](https://gist.github.com/d3vilguard/486c1c3b373e9010e95e6f8fd97d1d2a):
- Proton version: Proton Experimental - Bleeding Edge (also issue with latest 8)

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
Sniper scope glint is visible through objects (essentially like I'm wallhacking). Confirmed with Bleeding Edge and 8. Also map ZEEBRUGGE seems to have object issues also. 

## Reproduction
Have a sniper scoped behind object and see the glint when you are not supposed to.

**Comment by Blisto91** [comment]
@d3vilguard Hi there and thank you for the report. 
We've looked in to this scope glint issue and found it to be a game bug with the DX12 renderer. It also happens with the game on Windows.

**Comment by d3vilguard** [comment]
> @d3vilguard Hi there and thank you for the report. We've looked in to this scope glint issue and found it to be a game bug with the DX12 renderer. It also happens with the game on Windows.

Ahh, guess that one is on me for not researching in general if the game had the issue. With EA implementing anti-cheats left and right thought that if proton specific that it should be reported. Don't want this game to fallow BF V for such a minor issue.

**Comment by d3vilguard** [comment]
Everything Proton that is Wine 9 based (Proton 9 betas, Experimental since hitting 9, Proton 9) crashes the GPU moments before loading a multiplayer game (at the end of the loading screen). RX 6800, tried both mesa stable from arch and mesa-git. Three different installations of Arch, test in Windows, kernels tkg, zen, normal linux. Proton 8 has absolutely no problems loading matches and playing them.

**Comment by kisak-valve** [comment]
Hello @d3vilguard, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put them in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by d3vilguard** [comment]
My bad, @kisak-valve, lets turn this into somewhat proper report:
### System

- [gist](https://gist.github.com/d3vilguard/b8c989c98d7167d0c44560adf1ce3dee)
- mesa-minimal-git compiled per AUR guides in a clean chroot (llvm-minimal-git), also tested with stock mesa
- Kernel: Linux-TKG (BORE), also tested with linux-zen and linux

### Observed behavior
Using protons 9-beta(whatever was the version on March 17)/9-rc/9/Experimental-Bleeding, Battlefield 1:

- Loads
- Opens server browser
- Display loading videos while connecting to a match
- Right before transitioning from the loading screen to the menu where one selects a squad the game crashes
- Screen freezes at the loading menu
- GPU (and by the looks if journalctl) everything goes haywire 
- System will need a reboot

### Expected behavior

- Game should load a match and play it just fine - as it does with Proton 8.

### Other
Will actually go a step further, attaching **journalctl**. Game was opened at **17:25:45** and the reboot should be around **17:28**.

I've been having this issue for a few weeks now. 

- Reinstalled Arch, tried stock mesa/kernel
- Tried removing overclock, even underclocking

Prior to finding out that it runs with Proton 8 thought that it might be a hardware error.
- Did all kinds of GPU, CPU, memory controller stress tests to rule out hardware.
- Went as far to buy a brand new motherboard just in case.
- Tested the game in Windows prior to figuring out that it runs OK with Proton 8 - Windows ran the game fine.
- Figured out that Proton 8 runs the game independent of OC settings and software versions (mesa/kernel).

[steam-1238840.log](https://github.com/ValveSoftware/Proton/files/15222275/steam-1238840.log)
[journalctl.txt](https://github.com/ValveSoftware/Proton/files/15222278/journalctl.txt)

Any chance you guys have compiled early March or even February Proton 9 beta builds? Not in a mood to compile proton and it might be worth it to see if it started crashing after a particular 9 beta version?

Let me know if I can help with anything else!

**Comment by kisak-valve** [comment]
Looking at your journalctl log, there's a GPU reset which kwin has a bad time with and falls over.
Looking at your Proton log, you have both bleeding edge mesa and AMDVLK, and it isn't clear which is being used with the game. Can you test how the game behaves when it only has access to mesa/RADV or AMDVLK?

**Comment by d3vilguard** [comment]
> Looking at your journalctl log, there's a GPU reset which kwin has a bad time with and falls over. Looking at your Proton log, you have both bleeding edge mesa and AMDVLK, and it isn't clear which is being used with the game. Can you test how the game behaves when it only has access to mesa/RADV or AMDVLK?

Well it seems that I am mentally challenged and have absolutely no idea when amdvlk slipped into being installed. Nuking it out of existence does fix the crashing with Proton 9. I have absolutely no idea if 8 was using radv or amdvlk, no clue why 9 was crashing, but again only leaving ravd on the system fixes everything. Thank you, sincerely! 

p.s. if one is to do a "minimal" installation using `archinstall` amdvlk will get installed. That's how it got there.

**Comment by solenum** [comment]
How are you guys getting past the EA app?  For me it will just pop up for a moment, close and that's it, I can never get in-game.  Battlefield 5 runs fine, as does 4, etc.  It seems just this ones broken for me.

**Comment by Am0rphous** [comment]
I experience the exact same symptoms @solenum .

No luck. Here is my speccs and some stuff I've tried:

I use Kali Linux 6.6.9-amd64 with RTX 3080TI (545.29.02). I reinstalled the EA app in compatibility mode in Steam (proton experimental. I downloaded and "repaired" the game. The EA app opens but Battlefield1 as a black window before it quits it after a few seconds. Steam logs are located at `~/.local/share/Steam/logs` but I couldn't find anything worth highlighting while grepping on words like crash, failed, battlefield, error or EA...

I suspect the game stopped working after EA updated their app. Because after the auto update the game was broken when trying to open it.

**Comment by solenum** [comment]
Disregard my last, none of them work now and I just get 'the app timed out' when trying to start battlefield 4 etc.  I was literally playing last night :|

Edit;  Setting to proton 7 and letting the ea app install, and then setting back to proton experimental fixed bf4 for me, haven't tried that on 1 yet as I don't currently have it installed.

Double edit;  (bf4) got kicked by punkbuster after a few mins in a match :))

**Comment by Am0rphous** [comment]
It's suddenly working @solenum .

Im using latest EA software and run it in Proton 8.0-5 in Steam. Cheers!

**Comment by solenum** [comment]
You're right, after starting with Proton 7 to let the EA app install (like bf4 etc), the game seems to run and work with Proton 8.  Experimental breaks it.

**Comment by hb0nd** [comment]
the game is crashing on start for me, it starts the EA launcher fine, then shows a black window with the title "battlefield 1" for a few seconds, then crashes to the EA launcher which closes.

the log backtrace seems to start at `amd_ags_x64.dll`, so i tried running with `WINEDLLOVERRIDES="amd_ags_x64.dll=b" %command%`, but getting the same error.

system info: [gist](https://gist.github.com/RedAuburn/0f993c7edf53b71b3a61415498b17e7c)
log: [steam-1238840.log](https://github.com/user-attachments/files/15840947/steam-1238840.log)


update:
using proton 8 works!

**Comment by d3vilguard** [comment]
> How are you guys getting past the EA app? For me it will just pop up for a moment, close and that's it, I can never get in-game. Battlefield 5 runs fine, as does 4, etc. It seems just this ones broken for me.

um?? BF V doesn't run anymore due to anti-cheat?

For those of you that had trouble with the EA app. From all of my EA games I see that sometimes it just closes the first 1-2 attempts. Then it opens. If it starts giving trouble - verify game files. It if continues - see the ID of the game and nuke it's compatdata. That will make steam install the EA app again.

**Comment by solenum** [comment]
They've all been working for me since my last comment with Proton 8.0-5, sometimes the EA app stops working and I just verify game files and that seems to fix it, but aside from that no real issues.  If you're having trouble nuke your compatdata directory and try with Proton 8.0-5.

**Comment by alasky17** [comment]
@RedAuburn Edit #2: I just realized that the issue you are describing *should* be fixed with Experimental bleeding-edge branch.  This will be in "proper" experimental soon if you don't want to try out bleeding-edge.  I would still recommend saving your prefix just in case :)  The request below only applies if you still get a failure on bleeding-edge.

Could you please try WINEDLLOVERRIDES="amd_ags_x64=n" %command% with Proton Experimental and see if you still get the same failure with Proton 9.0-x?  If that succeeds, could you please get 2 logs using "PROTON_LOG=+amd_ags %command%"  -- one of the successes with amd_ags_x64=n and one of the failure with amd_ags_x64=b? 

Edit: It is probably a good idea to save your old prefix first so you can get back to your working state if this doesn't fix anything.

**Comment by alasky17** [comment]
The NVIDIA regression fix is now in regular Proton Experimental.  If anyone is still seeing failures that only happen on experimental-9 and are "fixed" on 8.0-5, please let me know :)  9.0-2 still has the breakage.

**Comment by gazben** [comment]
After the patch yesterday I see this error now:

![image](https://github.com/user-attachments/assets/5ab03083-c719-41c3-b423-38fcc0b5a190)



I treat this as a personal attack from EA...

**Comment by HonestPorcupine** [comment]
Same there. Just played in beginning of October without any problem
Decided to play again today, yeah...

**Comment by tjbbjt** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5047#issuecomment-2431200594

Getting the same, game won't start at all. This is an EA anti-cheat problem?

**Comment by tjbbjt** [comment]
@kisak-valve Do you need logs or anything for this current error? Or is this an EA issue not a steam compatibility issue?

**Comment by rcv11x** [comment]
Hello Valve or whoever reads this, I am a linux user and I just want to say that I love playing there but it is a shame that companies like EA suddenly update their anti-cheat and stop giving support to linux users, before battlefield 1 and 5 worked perfectly but since last year they updated the anti-cheat and we can not play, this kind of things should not be allowed and should do something about it, if not I wish you could put a solution or contact with such companies, in this case EA, thanks for reading me.

**Comment by Am0rphous** [comment]
I agree. This is outrageous.

2 months ago i asked in the forums about how many virtual systems are in use today. Since they banned virtualization cloud gaming should not exist, unless cloud vendors use a specific compiled binary or they somehow got their hardware fingerprint "whitelisted". I also requested info about cheating statistics, because this has to have been improved right? We would love to hear that the ban on VMs has resulted in a "sucess story".

Their ignored answering what i asked for and said "their development cycle was finished a long time a go, so no updates or policy changes - like adjustments to anti-cheat sytem will be made. In addition the **specific reasons are not publicly disclosed**".

So basically it's not for the player's own good - if it were, they'd be transparent and not secretive. Guess they're back to supporting closed source OSes for data harvesting. Correct me if I'm wrong.
